test/broadcasting.jl

```diff
@@ -1152,7 +1152,7 @@ end
         end
         df6 = copy(df2)
         df7 = copy(df3)
-        df4 = DataFrame(sin.(df1[1,1] .+ copy(df1[!, 1]) .+ Matrix(df2) ./ Matrix(df3)))
+        df4 = DataFrame(sin.(df1[1,1] .+ copy(df1[!, 1]) .+ Matrix(df2) ./ Matrix(df3)), names(df3))
```

Comment:
Here and below `:gennames` would also be more obvious.

Reply:
where I use `names` I test if the names from some concrete data frame are reused, I remove the explicit `[:x1, :x2]` and similar with `:auto`.